src/parameterizations/vertical/MOM_opacity.F90

```diff
@@ -309,12 +319,13 @@ subroutine opacity_from_chl(optics, sw_total, sw_vis_dir, sw_vis_dif, sw_nir_dir
         if (G%mask2dT(i,j) > 0.5) then
           if (multiband_vis_input) then
             SW_vis_tot = sw_vis_dir(i,j) + sw_vis_dif(i,j)
-          else  ! Follow Manizza 05 in assuming that 42% of SW is visible.
+          elseif (total_sw_input) then
```

Comment:
Should there be a final else with a `MOM_warning` call for the case where `multiband_vis_input` = .false. and `total_sw_input = .false.`?

Reply:
There is actually a `MOM_error(WARNING, ...)` call that already covers this case where both multiband_vis_input and total_sw_input are false about 45 lines earlier in the code (and outside of the do loops), along with logic so that we only get the warning message the first time this routine is called.